package.json

```diff
@@ -1,6 +1,5 @@
 {
 	"private": true,
-	"type": "module",
```

Comment:
as per
https://github.com/TypeStrong/ts-node/discussions/1337
waiting for
https://github.com/TypeStrong/ts-node/pull/1371

Reply:
This is not a change we want, the project should stay type:module

Reply:
Correct that is why I closed The PR I think https://github.com/TypeStrong/ts-node/pull/1371 will fix the issue.